control/controlclient/direct.go

```diff
@@ -1220,6 +1220,10 @@ func loadServerPubKeys(ctx context.Context, httpc *http.Client, serverURL string
 	if err != nil {
```

Comment:
Capitalize as you were writing a sentence

https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#comment-sentences

Reply:
Resolved just before push